### PR TITLE
SDK Server subscriptions work

### PR DESCRIPTION
**Pull request by robert-e-davidson3**

### CODE REVIEW

File: packages/augur-ui/src/modules/events/actions/listen-to-updates.ts [hunk 1]
```diff
@@ -62,6 +64,13 @@ const EVENTS = {
   [SubscriptionEventName.WarpSyncHashUpdated]: wrapLogHandler(
     handleWarpSyncHashUpdatedLog
   ),
+  [SubscriptionEventName.OrderEvent]: wrapLogHandler(
```
Comment (bthaile) [inline packages/augur-ui/src/modules/events/actions/listen-to-updates.ts]: UI listens for bulk zerox order events not individual order events there is too much chatter and causes too many getter calls. This event wire up in start up events collection.

`[SubscriptionEventName.BulkOrderEvent]: wrapLogHandler(handleBulkOrdersLog),`

Need to follow the same pattern as bulk order events.

**Review by bthaile — CHANGES_REQUESTED** [review]
Can these SDK server zerox order events be a bulk event. UI doesn't subscribe to individual zerox order events.

File: packages/augur-ui/src/modules/events/actions/listen-to-updates.ts (hunk 1 above)
Comment (robert-e-davidson3) [reply packages/augur-ui/src/modules/events/actions/listen-to-updates.ts]: Fixed!

**Review by pgebheim — APPROVED** [review]

File: packages/augur-ui/src/modules/events/actions/log-handlers.ts [hunk 2]
```diff
@@ -405,6 +405,14 @@ export const handleBulkOrdersLog = (data: {
   }
 };
 
+export const handleDBUpdatedZeroXOrders = (log: StoredOrder) =>
```
Comment (bthaile) [inline packages/augur-ui/src/modules/events/actions/log-handlers.ts]: if UI is processing bulk orders this isn't needed.

**Review by bthaile — CHANGES_REQUESTED** [review]
remove individual order event processing in UI. It should only be processing bulk zeroX order events

File: packages/augur-ui/src/modules/events/actions/log-handlers.ts (hunk 2 above)
Comment (pgebheim) [reply packages/augur-ui/src/modules/events/actions/log-handlers.ts]: Agreed, there should be zero changes to which events are generated or listened to.

As an optimization we can allow each SDK consumer to subscribe to the relevant events in the future.

Comment (robert-e-davidson3) [reply packages/augur-ui/src/modules/events/actions/log-handlers.ts]: Good catch. I missed this when I switched to bulk orders. Deleted.

**Review by bthaile — APPROVED** [review]
lgtm

**Pull request merged**